### PR TITLE
chore(db): add better dummy data for user, tag, tagToUser

### PR DESCRIPTION
**Pull request by tgrigoruk**
### Summary
Create better dummy data for `user`, `tag`, and `userToTag` entities.

- Made some small updates/fixes to the schema.
- Added 15 `user`s with complete and varied profile data.
- Added about 20 `tag`s
- `userToTag` are generated such that a user with higher ids have more interests.

**BLOCKING: #109 user interests endpoints**

### Retrospective
Defined relations according to https://orm.drizzle.team/docs/rqb#many-to-many
Also foreign key and FK actions described just below that.

### Testing Notes
The `.sql` and `json` files are auto-generated.

Might be good to have a giant list of tags (like 100s, in a separate file) for better testing (like with recommendations)

### Testing
Delete existing tables then start project as normal to recreate db. Check out the new entries with phpMyAdmin.

### CODE REVIEW

File: apps/api/src/models/schema.ts [hunk 1]
```diff
@@ -22,34 +22,36 @@ export const tag = mysqlTable('tag', {
   id: int('id').primaryKey().autoincrement(),
   createdAt: timestamp('createdAt').notNull().$defaultFn(() => new Date()),
   creatorId: int('creatorId'),
```
Comment (tgrigoruk) [inline apps/api/src/models/schema.ts:24]: `creatorId` should reference a `user`!

File: apps/api/src/models/schema.ts [hunk 2]
```diff
@@ -22,34 +22,36 @@ export const tag = mysqlTable('tag', {
...
+  text: varchar('text', { length: 50 }).notNull().unique(),
```
Comment (tgrigoruk) [inline apps/api/src/models/schema.ts:25]: Is 50 chars enough for a tag? Too much?

Comment (FoRVaiS) [reply apps/api/src/models/schema.ts:25]: 50 is fine

**Review by FoRVaiS — APPROVED** [review]

**Comment by FoRVaiS** [comment]
Once we start saving data onto a production database, we should avoid dropping the sql files.

Drizzle (and other ORMs) generate increment sql files so that existing production databases can be updated without losing data.

**Pull request merged**